Change Notification's private preview is over, and all users must instead use Monitor's direct integration with Resource Graph. (#2)
CawaMS/EnableChangeNotifications: ChangesIncurCost.md

> **Update 2024-04-11: This page is left as a reference for how Change Analysis Change Notification Preview users could use change data for setting up alerts with Azure Monitor. We recommend all Azure users to use Azure Monitor's native support for querying the Resource Graph for alert queries on resource change data. See the [annoucement blog](https://techcommunity.microsoft.com/t5/azure-observability-blog/query-azure-resource-graph-from-azure-monitor/ba-p/3918298) and [official documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/azure-monitor-data-explorer-proxy) for more information.**

Reviewer: ```suggestion
### Update 2024-04-11
This page serves as a reference for how Change Analysis Change Notification private preview users could use change data for setting up alerts with Azure Monitor. We recommend using Azure Monitor's native support for querying the Resource Graph for resource change data alerts. 

For more information, see the [announcement blog](https://techcommunity.microsoft.com/t5/azure-observability-blog/query-azure-resource-graph-from-azure-monitor/ba-p/3918298) and [official documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/azure-monitor-data-explorer-proxy).

--- 

```
